e2e/native/BpkSwitch.spec.js

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I think the detox specs should live with the component

Reply:
Yeah that seems reasonable imo